apps/zui/docs/features/Packet-Captures.md

```diff
@@ -1,7 +1,3 @@
----
-sidebar_position: 2
```

Comment:
I dropped these `sidebar_position` directives in all these Features articles since that way they're automatically sorted alphabetically.